StortSpelProjekt/Game/src/Gamefiles/Components/EnemyComponent.cpp

```diff
@@ -30,11 +32,21 @@ void component::EnemyComponent::Update(double dt)
         m_pParent->GetComponent<component::Audio3DEmitterComponent>()->UpdateEmitter(L"OnGrunt");
         m_pParent->GetComponent<component::Audio3DEmitterComponent>()->Play(L"OnGrunt");
     }
+
+    if (strcmp(m_pParent->GetName().substr(0, 11).c_str(), "enemySpider") == 0)
```

Comment:
This if statement is unneccessary!
The sound will only be updated if it has been loaded to the component